### PR TITLE
[14.0] project_scrum: migration to V14

### PR DESCRIPTION
**Pull request by alan196**
### Proposed changes

Migration PR for module project_scrum.

### Types of changes

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Migration Update

### Checklist

- [x] I have read the CONTRIBUTING doc
- [x] I have signed the CLA
- [x] Lint and unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have updated the necessary documentation

### Further comments

This PR contains the commit of PR https://github.com/OCA/project-agile/pull/28

### CODE REVIEW

File: project_scrum/views/project_project.xml [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+            <xpath expr="//div[@name='options_active']" position="inside">
+                <div>
+                    <field name="use_scrum" class="oe_inline" />
+                    <label for="use_scrum" string="Use Scrum" />
+                </div>
+            </xpath>
+        </field>
+    </record>
+
+    <record id="act_project_tasks" model="ir.actions.act_window">
```
Comment (ernestotejeda) [inline project_scrum/views/project_project.xml:34]: XML shortcut tags <act_window> are no longer valid, and must be replaced by its full definition through <record> tag.

**Review by ernestotejeda — COMMENTED** [review]
Hi @alan196 thanks for the migration.

**Comment by elvise** [comment]
@pedrobaeza any chance to merge ?

**Comment by pedrobaeza** [comment]
There's a comment unattended. @alan196 can you please check?

**Comment by alan196** [comment]
The comment was attended I forgot to comment that it was solved @pedrobaeza @ernestotejeda

**Pull request merged**